content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,774 @@
+
+## Configuring Pod Security
+    
+### Pod Security Standards
+    
+In order to use Pod Security we first need to understand [Pod Security Standards](/docs/concepts/security/pod-security-standards/). These standards define three different policy levels that range from permissive to restrictive. These levels are as follows:
+   * Privileged — open and unrestricted
+   * Baseline — Covers known privilege escalations while minimizing restrictions
+   * Restricted — Highly restricted, hardening against known and unknown privilege escalations. May cause compatibility issues
+
+Each of these policy levels define which fields are restricted within a pod specification and the allowed values. Some of the fields restricted by these policies include:
+   * `spec.securityContext`
+   * `spec.hostNetwork`
+   * `spec.volumes[*].hostPath`
+   * `spec.containers[*].securityContext`
+
+Policy levels are applied via labels on Namespace resources, which allows for granular per-namespace policy selection. The AdmissionConfiguration in the api server can also be configured to set cluster-wide default levels and exemptions.
+
+### Policy modes
+
+Policies are applied in a specific mode. Multiple modes (with different policies) can be set on the same namespace. Here is a list of modes:
+   * `enforce` — Any Pods that violate the policy will be rejected
+   * `audit` — Violations will be recorded as an annotation in the audit logs, but don't affect whether the pod is allowed.
+   * `warn` — Violations will send a warning message back to the user, but don't affect whether the pod is allowed.
+
+In addition to modes you can also pin the policy to a specific version for example v1.22. Pinning to a specific version allows the behavior to remain consistent as the policy definition changes over Kubernetes releases. If pinning to a specific Pod Security Standard version in a Kubernetes release you should take the time to understand the Kubernetes [supported version policy](/releases/version-skew-policy/#supported-versions).
+
+## Hands on demo
+
+### Prerequisites
+
+- [kind CLI installed](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/getting-started/installation) container runtime & CLI
+
+### Deploy a kind cluster
+
+```shell
+kind create cluster --image kindest/node:v1.23.0
```

Comment:
I had the same issue and built one too (then updated PR not to use my local tag). I have it listed in my "steps before hold can be lifted" to include changing to the latest `sha256` tag. With the PR targeting `main` it can merge at anytime, even post-release.